src/bin/pgcopydb/schema.c

```diff
@@ -423,11 +423,34 @@ schema_list_extensions(PGSQL *pgsql, DatabaseCatalog *catalog)
 	SourceExtensionArrayContext parseContext = { { 0 }, catalog, false };
 
 	char *sql =
+		"with recursive fk_constraints as ( "
+		"     select * from pg_constraint where contype = 'f' and conrelid <> confrelid "
```

Comment:
Forgot to mention: `SELECT *` is considered harmful in code. Interactive use is another story. Please expand the column names you need. Hint: we don't know what version of Postgres the code is going to connect to at run-time.

Reply:
done